content/en/docs/tasks/traffic-management/istioctl-describe/index.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Understanding your mesh with with istioctl describe [Experimental]
+description: Shows you how to use istioctl to determine the traffic management applied to a pod.
+weight: 90
+keywords: [traffic-management, istioctl, debugging]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+In Istio 1.3, we included the experimental `describe pod` sub-command for `istioctl`. We designed this tool to help find and
+understand the configuration that impacts the traffic to a pod.  This task shows you how
```

Comment:
This is just one part of the scope, isn't it?  Istio supports more than traffic and this command isn't just to troubleshoot traffic related issues.

Reply:
It would be help user understand if the service and associated pod is ready for mesh, and the type of secure/plain communication?